Source/Scene/Cesium3DTileset.js

```diff
@@ -788,19 +816,57 @@ define([
         }
     }
 
-    function getScreenSpaceError(geometricError, tile, frameState) {
+    var scratchPositionNormal = new Cartesian3();
+
+    function updateDynamicScreenSpaceError(tileset, frameState) {
+        var camera = frameState.camera;
+        var positionCartographic = camera.positionCartographic;
+        var height = positionCartographic.height;
+
+        // The range where the density starts to lessen
```

Comment:
If the max height is needed (not sure why, we should be thinking in general 3D for 3D Tiles), I think it can be computed pretty easily for spheres.  OBBs could be harder, but if they are too hard for now; we could approximate them with a sphere for this computation.

For spheres, project the point onto the surface using the geodetic surface normal.  Subtract that distance from the radius, then go from the surface point along the geodetic normal to (radius - that